examples/ui/text_debug.rs

```diff
@@ -120,11 +127,30 @@ fn infotext_system(commands: &mut Commands, asset_server: Res<AssetServer>) {
     });
 }
 
-fn change_text_system(mut query: Query<(&mut Text, &TextChanges)>) {
+fn change_text_system(
+    time: Res<Time>,
+    diagnostics: Res<Diagnostics>,
+    mut query: Query<(&mut Text, &TextChanges)>,
+) {
     for (mut text, _text_changes) in query.iter_mut() {
```

Comment:
If we're not using `_text_changes` we could just replace it with an `_`, but that's not something that important.

Reply:
Obviously its currently valid as it is

Reply:
Alternatively, `Query<&mut Text, With<TextChanges>>` would allow `for mut text in query.iter_mut() {}`

Reply:
Good point haha

Reply:
I switched it to a `With` query instead.